code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -658,6 +658,16 @@
 	construction_time = 120
 	category = list("Cyborg Upgrade Modules")
 
+/datum/design/borg_upgrade_ashplating
+	name = "Cyborg Upgrade (Ash Storm Plating)"
+	id = "borg_upgrade_ashplating"
+	build_type = MECHFAB
+	build_path = /obj/item/borg/upgrade/ashplating
+	req_tech = list("plasmatech" = 4, "materials" = 4, "engineering" = 4)
```

Comment:
This is meant to be a **medium** effort item. Basically along the lines of "If Scientists are competent, they can make this item on request". I fully intend this to be something you could get by collecting plasma from xenobio or engineering.

This line of my mining related additions (satchel of holding, the corresponding borg version, this item) are intended to be obtainable in the early stages of the round if you specifically seek them out.